docs/DeveloperGuide.md

```diff
@@ -607,7 +605,165 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+### Adding a delivery
```

Comment:
It would be nice to add a title
### Testing Delivery Commands
before this section!